src/test/java/com/currencyexchange/business/ExchangeRateUpdateServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package com.currencyexchange.business;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.currencyexchange.provider.ExchangeRateProvider;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class ExchangeRateUpdateServiceTest {
+
+  @Mock
+  private ExchangeRateProvider firstFetcher;
```

Comment:
It is better to choose a name according to the type of the object - exchangeRateProvider. If you have several - exchangeRateProviderFirst (If the differences are not important)